n3fit/src/evolven3fit_new/__init__.py

```diff
@@ -0,0 +1 @@
+from validphys._version import __version__, __version_tuple__
```

Comment:
Why do we need a `__version_tuple__`?

Reply:
I saw that it was for free with the dynamic version package so I decided to add it as well.